rasa/nlu/tokenizers/whitespace_tokenizer.py

```diff
@@ -25,7 +22,7 @@ class WhitespaceTokenizer(Tokenizer, Component):
         "intent_tokenization_flag": False,
         # Symbol on which intent should be split
         "intent_split_symbol": "_",
-        # text will be tokenized with case sensitive as default
+        # Text will be tokenized with case sensitive as default
         "case_sensitive": True,
```

Comment:
this was added for crf extractor. Technically it should be part of the featurizer. If we deprecate old crf extractor, I think, we should remove this one

Reply:
sounds good, will add it to the list